projects/fal/src/fal/cli/deploy.py

```diff
@@ -219,5 +220,13 @@ def valid_auth_option(option):
         help="Deployment strategy.",
         default="recreate",
     )
+    parser.add_argument(
+        "--no-scale",
+        action="store_true",
+        help=(
+            "Use min_concurrency/max_concurrency/max_multiplexing from previous "
+            "deployment of application with this name."
+        ),
+    )
```

Comment:
curious about these:
1. What will happen if I pass both concur config and no-scale? will no scale take precedence?
2. what happens if the app is not deployed previously and I did not pass any concur config? will it fail?

Reply:
1. You mean if you have concur in the app? no-scale will ignore it and will use previous deploymen'ts concur settings.

2. it will deploy with the ones in the App code.

Very good questions, let me try clarifying that in the message.